app/presenters/publishing_api/person_presenter.rb

```diff
@@ -35,8 +35,7 @@ def details
       details_hash = {}
 
       if item.image&.all_asset_variants_uploaded? && item.image&.url(:s465)
-        raise "person image path should not use carrierwave path" if item.image.url(:s465).include?("carrierwave-tmp")
-
+        logger.info("PersonPresenter: Person of ID##{item.id} has image with url '#{item.image&.url(:s465)}'")
```

Comment:
I was imagining we'd log only if the invalid carrierwave temporary path is used? I'm not sure the output is that interesting otherwise. Also in that case I think we'd log at the error level.